examples/chatter/talker.cc

```diff
@@ -25,12 +25,8 @@
 class MinimalPublisher : public rclcpp::Node {
  public:
   MinimalPublisher() : Node("minimal_publisher"), count_(0) {
-    int callback_period_ms = 500;
-    declare_parameter("callback_period_ms");
-    try {
-      callback_period_ms = get_parameter("callback_period_ms").as_int();
-    } catch (const rclcpp::exceptions::ParameterNotDeclaredException&) {
-    }
+    declare_parameter("callback_period_ms", 500);
+    auto callback_period_ms = get_parameter("callback_period_ms").as_int();
```

Comment:
In foxy this fires an exception if the node is not started with the param defined. Is this different in Huble?

The idea was to have this by default set to 0.5s and set to something much smaller for tests to take less time.

Reply:
The second parameter to declare_parameter is the default value.